src/BaseStyles.js

```diff
@@ -81,7 +81,6 @@ pre[class*="language-"] {
   color: var(--code-color, black);
   text-shadow: 0 1px white;
   text-shadow: var(--http-code-snippet-code-text-shadow, 0 1px white);
-  font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
```

Comment:
We use `--arc-font-code-family`  to define code fonts. This is still the code view so this should start as a variable at least.